src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -119,13 +119,24 @@ public User getUser(String email) {
   */
   public Set<String> getUsers()
   {
-	Set<String> users = new HashSet<>();
-	Query query = new Query("Message");
-	PreparedQuery results = datastore.prepare(query);
-	for(Entity entity: results.asIterable())
-	{
-	  users.add((String)entity.getProperty("user"));
-	}
-	return users;
+    Set<String> users = new HashSet<>();
+	
+    /*adding users who have posted a message*/
+    Query query = new Query("Message");
+    PreparedQuery results = datastore.prepare(query);
+    for(Entity entity: results.asIterable())
+    {
+      users.add((String)entity.getProperty("user"));
+    }
+	
+    /*adding users who have added about me*/
+    Query query2 = new Query("User");
```

Comment:
Thank you for your comments. I will do the renaming. about the order of two queries, at first, I thought it matters because for users who have posted both aboutMe and messages, if I did Query("Message") first I could not get the property "aboutMe" (Message does not have that property), and I thought if I add it to HashSet it will not add it again when I query "User" that actually has "aboutMe" property. However, I just realized it adds them twice because the string that get hashed are different (one is user, and other is same user+aboutMe), and it doesn't matter what the order is. So there is a bug in my code, and I should fix it! Thanks for helping me see this bug, I missed it!